### DISCUSSION TITLE
"And" & "Or" trigger combiners

### DISCUSSION THREAD

**Discussion by FabioOG** [redeclipse/base#1150] [Ideas and Feature Requests]
Triggers are cool, but the problem is we can't make 2 different ways of opening a door or it will bug.
I think we could need some kind of trigger combiner to link 2 triggers using logic to get more than 1 way to trigger something

**Comment by qreeves** [comment]
Yes, I understand. Will look into this in the future.

**Comment by IceflowRE** [comment]
Technically you can already do this with cubescript.

**Comment by cloudziq** [comment]
yes, but I think we need more trigger types by default, problem with map scripts is that these scripts get 'flushed' (removed) after every map save
By more types I mean something like Goldenbrain's idea, also timed triggers (working after certain time), or easy ability to make combined triggers (like player need to use 5 triggers in order to unlock something)

**Comment by FabioOG** [comment]
Hmm didn't think to that type of multi-key but could be cool in some race map
